### PR TITLE
coreos-base/coreos: add sys-devel/gettext

### PR DESCRIPTION
**Pull request by tormath1**
This Flatcar dependency needs to be now explicitly pulled in the OS since this commit: https://github.com/flatcar-linux/coreos-overlay/pull/1724/commits/4a06200e9d92dc8066425cfc18dd6b7c5aa99131

Signed-off-by: Mathieu Tortuyaux <mtortuyaux@microsoft.com>

<hr>

- [x] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)
- [x] Inspected CI output for image differences: `/boot` and `/usr` size, packages, list files for any missing binaries, kernel modules, config files, kernel modules, etc.
```
14:05:51  +/usr/bin/envsubst
...
14:05:51  +/usr/bin/gettext
14:05:51  +/usr/bin/gettext.sh
14:05:51  +/usr/bin/gettextize
```
- [x] CI: http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/6491/cldsv/ (except for dev-container but seems unrelated)

Closes: https://github.com/flatcar-linux/Flatcar/issues/849

### CODE REVIEW

**Review by pothos — APPROVED** [review]

File: coreos-base/coreos/coreos-0.0.1.ebuild [hunk 1]
```diff
@@ -175,6 +175,7 @@ RDEPEND="${RDEPEND}
 	sys-block/parted
 	sys-boot/efibootmgr
 	sys-cluster/ipvsadm
+	sys-devel/gettext
```
Comment (krnowak) [inline coreos-base/coreos/coreos-0.0.1.ebuild:178]: I'd say it's not obvious why we would want gettext to be in production images, so could you add a comment about it being added for `envsubst` binary that people found useful?

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]
Could we also add a entry in the `updates`?

**Comment by tormath1** [comment]
@sayanchowdhury @krnowak thanks for the review - comments have been addressed. :)

**Comment by krnowak** [comment]
> ```
> 14:05:51  +/usr/bin/envsubst
> ...
> 14:05:51  +/usr/bin/gettext
> 14:05:51  +/usr/bin/gettext.sh
> 14:05:51  +/usr/bin/gettextize
> ```

Hmm, do we want to put the last three files into `INSTALL_MASK`?

**Comment by tormath1** [comment]
> > ```
> > 14:05:51  +/usr/bin/envsubst
> > ...
> > 14:05:51  +/usr/bin/gettext
> > 14:05:51  +/usr/bin/gettext.sh
> > 14:05:51  +/usr/bin/gettextize
> > ```
> 
> Hmm, do we want to put the last three files into `INSTALL_MASK`?

Sure we could, there are also other binaries / files but I'm not sure which ones we can drop and which ones we should not: http://jenkins.infra.kinvolk.io:8080/job/os/job/board/job/image-matrix/2143/consoleFull

**Review by sayanchowdhury — APPROVED** [review]

**Comment by krnowak** [comment]
> > > ```
> > > 14:05:51  +/usr/bin/envsubst
> > > ...
> > > 14:05:51  +/usr/bin/gettext
> > > 14:05:51  +/usr/bin/gettext.sh
> > > 14:05:51  +/usr/bin/gettextize
> > > ```
> > 
> > 
> > Hmm, do we want to put the last three files into `INSTALL_MASK`?
> 
> Sure we could, there are also other binaries / files but I'm not sure which ones we can drop and which ones we should not: http://jenkins.infra.kinvolk.io:8080/job/os/job/board/job/image-matrix/2143/consoleFull

Yeah, I guess we will need to pore over our INSTALL_MASK eventually to see what needs to be dropped or added. Let's take this in for now.

**Review by krnowak — APPROVED** [review]

**Comment by tormath1** [comment]
cherry-picked to:
- flatcar-3227
- flatcar-3277
- flatcar-3346

**Pull request merged**